endToEndTests/test/queries/oneRowTwoUnalignedSequences.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Same here. It would help to find the test. But on thinking about it: maybe we could put the test files now in folders. So those which test the endpoints each in one, and then the filters in another. But this can also be done in another PR.